scene/main/viewport.h

```diff
@@ -204,16 +204,31 @@ class Viewport : public Node {
 	Camera2D *camera_2d = nullptr;
 	Set<CanvasLayer *> canvas_layers;
 
+#ifdef TOOLS_ENABLED
+	struct Camera2DOverride {
+		bool enabled = false;
+		ObjectID previous_camera = ObjectID();
```

Comment:
`ObjectID` is default initialized so you don't need to assign a new one here (same below).

Reply:
Ah, good spot! Thanks, fixed now. 🙂